CMake/FindBlas.cmake

```diff
@@ -0,0 +1,902 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
```

Comment:
If this is already part of a CMake release, we actually don't need to include it here---we can assume it's already available.  (That's true for `FindLapack.cmake` and `FindARPACK.cmake` too.)

Reply:
I don't think it's true for FindARPACK, but it should be for BLAS and LAPACK.